workspaces/sandbox/README.md

```diff
@@ -50,3 +50,18 @@ To generate knip reports for this app, run:
 ```sh
 yarn backstage-repo-tools knip-reports
 ```
+
+## Local Frontend Setup (provisional)
+The Sandbox plugin uses Red Hat SSO to authenticate users accessing the Sandbox backend. This section explains how to configure your local RHDH Sandbox UI to connect with Red Hat SSO and the Sandbox backend (currently, only the staging environment is supported).
+
+1. `cd workspaces/sandbox`
+2. `make push-plugin` (note that, currently, master is broken, so you need to skip this step)
```

Comment:
I think we can remove this. And just assume that it will be fixed/working very soon :) 
```suggestion
2. `make push-plugin`
```